### PR TITLE
scrapy sends data individually

### PR DESCRIPTION
**Pull request by ArtemKech**
# Changes:
Just changed `pipelines.py` file. I got rid of the part where _Scrapy_ was sending an array of articles at the end of the spiders' scraping 
Also added name of the article to the terminal's output for error checking  
Made the `server` to reject the articles that already exist in the `db`. No more duplicate articles!

### CODE REVIEW

File: scraper/newsscrapper/pipelines.py [hunk 1]
```diff
@@ -17,9 +17,7 @@ def __init__(self):
     
     def process_item(self, item, spider):
         self.json.append(item)
```
Comment (AardhynLavender) [inline scraper/newsscrapper/pipelines.py:19]: is this member this still needed?

**Review by AardhynLavender — CHANGES_REQUESTED** [review]
Looks like each call to `process_item` sends the accumulated articles rather than just the one being processed. 
Did this result in duplicated articles in your tests?

image

image

image

Should this be something like

```python
json_str: str = json.dumps(item, default=serialize_item)
article: dict = json.loads(json_str)
```

serialize just the acticl

**Review by gpseal — APPROVED** [review]
Good simple solution, will allow more flexibility when posting scraped data, nice job.

**Comment by ArtemKech** [comment]
My bad @AardhynLavender. Haven't noticed it doing this. I will fix that. Thanks for spotting this!

**Review by AardhynLavender — APPROVED** [review]
Epic

**Review by glwatkins — APPROVED** [review]
Fantastic, good additions to get the website functioning smoother.

**Pull request merged**